docs/state.md

```diff
@@ -478,8 +478,12 @@ WindowStore
       percentage: number,        // i.e., 0, 1, ... 100
       publisherURL: string,      // publisher site, e.g., "https://wikipedia.org/"
       faviconURL: string         // i.e., "data:image/...;base64,..."
+    } ],
+    options: {
```

Comment:
it would be useful to give this a more descriptive name, like `synopsisOptions`, because this property gets merged into ledgerData in js/components/frame.js